src/pal/src/arch/arm64/exceptionhelper.S

```diff
@@ -15,7 +15,9 @@ LEAF_ENTRY ThrowExceptionFromContextInternal, _TEXT
 #ifdef HAS_ASAN
         // Need to call __asan_handle_no_return explicitly here because we re-intialize SP before
         // throwing exception in ThrowExceptionHelper
+        stp x0, x1, [sp]
```

Comment:
This is missing pre-decrement of sp (and the restoration post-increment).

Reply:
I can also see that I've missed the same issue in the similar fix you did some time ago.

Reply:
I've fixed it, thank you!